src/Provider/GoogleUser.php

```diff
@@ -2,6 +2,8 @@
 
 namespace League\OAuth2\Client\Provider;
 
+use Google\Client;
+
```

Comment:
Doesn't seem to be used.

Reply:
That's because you likely don't have the dependency installed; it's used in the new method and if it doesn't exist and you call it, you get the runtime error.